### ISSUE TITLE
Suggestion: More verbose scenery library errors

### ISSUE THREAD

**Issue by cristi-neagu** [albar965/littlenavmap#655]
Hello,

After scanning my sim folders, LNM reports that it found errors with the files, in this case with non-existent files. I am pretty much certain that this is not a false flag and LNM is correct about the errors, but I wish I would know how to correct them. I've looked though all the files i could think of and i cannot find any mentions of those files. As such, could LNM report which file it scanned that told it those scenery files should exist?

Thanks.

**Comment by albar965** [comment]
What simulator? What files are reported? Can you copy and paste the error message or a screenshot?

**Comment by cristi-neagu** [comment]
![image](https://user-images.githubusercontent.com/30549178/120181217-a511d880-c204-11eb-88cd-140c912b2674.png)
Like i said, i didn't consider this information relevant since the errors are, i think, genuine.

**Comment by albar965** [comment]
These entries are a part of the `scenery.cfg` (or one of the other numerous files P3D uses) and the folders in these files do not exist. Maybe an add-on was not uninstalled completely.

I can try to add the referring file (`scenery.cfg`, `add-on.xml`, `add-ons.cfg` or others) to the message to make it more clear. Have to see how much effort this is.

Or omit the message completely. The simulator is very tolerant against all kind of inconsistencies.

**Comment by cristi-neagu** [comment]
Yes, I had a problem where P3D completely broke and i had to scrub it from my system. Obviously, i missed some bits, which is why it would be incredibly helpful if LNM would say where it found references to those missing files, as i have been through all the obvious files and i cannot find any reference of them. I also tried using a couple of addon managers hoping that they would find those references, and they weren't able to. So the only software that sees them is LNM. Again, it's not an issue with LNM, it's just the only one that sees them.

Either way, it's a nice to have, so i'm sure there's other more important things taking precedence.

Thank you.
